boards/Makefile.common

```diff
@@ -138,7 +138,7 @@ doc: | target
 lst: target/$(TARGET)/release/$(PLATFORM).lst
 
 .PHONY: release
-release: target/$(TARGET)/release/$(PLATFORM).bin
+release: target/$(TARGET)/release/$(PLATFORM).elf
```

Comment:
@ppannuto  How do we keep the .elf extension file around? The elf file is already there (just without an extension), but I think we want this as before since a .bin file is what we need to flash on the board in many? all? cases.

Reply:
In this configuration, I get both .bin and .elf as output with proper suffixes, so I prefer it. Some programs that work with .elf require the suffix

Reply:
I'm not immediately sure why the .bin file would get generated after this change.